dlp/snippets/deid.py

```diff
@@ -915,6 +917,135 @@ def write_data(data: types.storage.Value) -> str:
 # [END dlp_deidentify_date_shift]
 
 
+# [START dlp_deidentify_time_extract]
+import csv  # noqa: F811, E402, I100
+from datetime import datetime  # noqa: F811, E402, I100
+from typing import List  # noqa: F811, E402
+
+import google.cloud.dlp  # noqa: F811, E402
+
+
+def deidentify_with_time_extract(
+    project: str,
+    date_fields: List[str],
+    input_csv_file: str,
+    output_csv_file: str,
+) -> None:
+    """ Uses the Data Loss Prevention API to extract time part
+        data from a table.
```

Comment:
from a CSV file*

the underlying api uses a table, yes, but i think mentioning table here where there is no table object being passed into this function makes it harder to understand. you could mention that the CSV file is converted to a table input for DLP, and after deidentification, it is re-converted back into a CSV output file.

Reply:
Thank you for noticing that out. I have made required changes, please check.